src/module.c

```diff
@@ -222,8 +242,14 @@ int TSDB_info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
         series = RedisModule_ModuleTypeGetValue(key);
     }
 
-    RedisModule_ReplyWithArray(ctx, 7*2);
+    RedisModule_ReplyWithArray(ctx, 10*2);
 
+    RedisModule_ReplyWithSimpleString(ctx, "totalSamples");
+    RedisModule_ReplyWithLongLong(ctx, getTotalSample(series->chunks));
+    RedisModule_ReplyWithSimpleString(ctx, "memoryUsage");
```

Comment:
@K-Jo I think we can drop the memory usage from here since we have a separate command for it.

Reply:
IMHO, since we size changes rapidly when series is active and are already at O(chunks) time complexity for `number of samples`, it makes sense to provide it in info.

Reply:
@danni-m I don't see any issue with keeping it here as well as long as it calls the same logic.  I furthermore think it might be interesting to understand how much your timeseries + all it's downsampled rules occupy as a second info parameter. (no need to add this to this PR however)
@ashtul the `number of samples` should be in ts.info, but this isn't part of the PR right?

Reply:
```RedisModule_ReplyWithSimpleString(ctx, "totalSamples");```
```RedisModule_ReplyWithLongLong(ctx, getTotalSample(series->chunks));```